homeassistant/components/moscow_yandex_transport/sensor.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+'''
```

Comment:
Please use double quotes for docstrings. Black code formatter should take care of this.